code-samples/serving/grpc-ping-go/README.md

```diff
@@ -32,11 +32,8 @@ for production containers.
   Replace `{username}` with your Docker Hub username then run the commands:
 
   ```bash
-  # Build the container on your local machine.
-  docker build --tag "{username}/grpc-ping-go" .
-
-  # Push the container to docker registry.
-  docker push "{username}/grpc-ping-go"
+  # Build and push the container on your local machine.
+  docker buildx build --platform linux/arm64,linux/amd64 -t "{username}/grpc-ping-go" --push .
```

Comment:
The new command failed for me with the following output:

```console
[+] Building 0.0s (0/0)                                                                                  
ERROR: multiple platforms feature is currently not supported for docker driver. 
Please switch to a different driver (eg. "docker buildx create --use")
```

whereas the original commands worked without issue. If we're going to update it to use `buildx`, we'll need some more set up info since it doesn't work out of the box.

Reply:
I think adding a link to the [buildx](https://docs.docker.com/build/buildx/install/) should be enough.

Reply:
Personally, I think I'd rather leave the default as-is and provide a link to buildx for those who need to do a multiarch build (which won't be everyone)

Reply:
If you let people use buildx from the beginning, the container image will run on many people's PCs.
You will not have to worry about CPU architecture errors.
Therefore, buildx is also a good choice.